app/assets/stylesheets/internal/layout.scss

```diff
@@ -1,5 +1,6 @@
-@import '../variables';
 @import '../crayons';
+@import '../scaffolds';
```

Comment:
Imported the `scaffolds` (since those are the styles applied to broadcasts) purely so that we can "preview" a broadcast from the internal pages without duplicating CSS in two places.